src/page-building/blog/blog-page.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+
+import styled from 'styled-components';
+import { ButtonText, SendButton } from '../../components/form/controls';
+import { RightArrowIcon } from '../../components/career-form/icons/right-arrow';
+import { useI18next, useTranslation } from 'gatsby-plugin-react-i18next';
+import { GatsbyImage, getImage } from 'gatsby-plugin-image';
+import { Layout } from '../../components/layout/layout';
+import { SectionHeader } from '../../new-components/section-header/section-header';
+import { useLocaleFormat } from '../../new-components/i18n-helpers';
+import { Teaser } from '../../new-components/teaser/teaser';
+
+interface BlogPageProps {
+  posts: any[];
+}
+
+const OnlyEnglishNotice = () => {
+  const { changeLanguage } = useI18next();
+  const switchToEnglish = async (event) => {
+    event.preventDefault();
+    await changeLanguage('en');
+  };
+
+  return (
+    <div>
+      <SendButton onClick={switchToEnglish}>
+        <ButtonText>Zum Blog</ButtonText> <RightArrowIcon />
+      </SendButton>
+    </div>
+  );
+};
+
+const ExtraWideContainer = styled.div`
+  /**
+    Our layout has multiple tracks and we assign the wide one here.
+   */
+  && {
+    //grid-column: wide-start/wide-end;
+  }
+`;
+
+const BlogTeaserGrid = styled.div`
+  display: grid;
+  gap: 24px;
+  justify-items: stretch;
+  // we fetch teaser of size 600px
+  // this means we can search for a column size of 300px - 12px (half gap) = 288px
+  // to auto fit our teasers
+  grid-template-columns: repeat(auto-fill, minmax(288px, 1fr));
+`;
+
+const BlogHeader = styled(SectionHeader)`
+  margin-bottom: 80px;
+`;
+
+export const BlogPage = ({ posts }: BlogPageProps) => {
+  const { t } = useTranslation();
+  const { language } = useI18next();
+  const dateFormatter = useLocaleFormat('dd. MMMM yyyy');
+
+  const BREADCRUMB = [
+    { pathname: '/', label: 'Satellytes' },
+    { pathname: '/blog', label: t('navigation.blog') },
+  ];
+
+  return (
+    <Layout light showLanguageSwitch={false} breadcrumb={BREADCRUMB}>
+      <ExtraWideContainer>
+        <BlogHeader headline={t('navigation.blog')}>
+          {t('blog.info')}
+        </BlogHeader>
+      </ExtraWideContainer>
+
+      <ExtraWideContainer>
+        {language != 'en' && <OnlyEnglishNotice />}
```

Comment:
This looks out of place:

![image](https://user-images.githubusercontent.com/7970458/146203819-3f4795cb-1b20-4359-9627-de79b7281bab.png)

Reply:
oh I did not check this for a while.

Reply:
fixed, see https://satellytescommain21751-gkblog.gtsb.io/de/blog/

![Screen Shot 2021-12-15 at 16 33 58](https://user-images.githubusercontent.com/1701755/146216119-8e8a7f2e-fa0d-4a06-bf16-84d31539675c.png)